packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -149,7 +156,9 @@ export async function apiFetchCustomer(): Promise<Customer> {
   );
 }
 
-export async function apiFetchPlanUpgradeEligibility(planId: string) {
+export async function apiFetchSubscriptionChangeEligibility(
+  planId: string
+): Promise<SubscriptionEligibilityResult> {
```

Comment:
This doesn't appear to be the correct type. Looks like the API returns an object with one property, `eligibility`. Not sure if we have a type for this yet.